tests/apps/courses/test_templates_blogpost_detail.py

```diff
@@ -153,3 +154,102 @@ def test_templates_blogpost_detail_cms_published_content_opengraph(self):
         self.assertContains(
             response, '<meta property="og:image:height" content="500" />'
         )
+
+    def test_templates_blogpost_detail_meta_description(self):
+        """
+        The blogpost meta description should show meta_description placeholder if defined
+        """
+        blogpost = BlogPostFactory()
+        page = blogpost.extended_object
+
+        title_obj = page.get_title_obj(language="en")
+        title_obj.meta_description = "A custom description of the blog post"
+        title_obj.save()
+
+        page.publish("en")
+
+        url = blogpost.extended_object.get_absolute_url()
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+        self.assertContains(
+            response,
+            '<meta name="description" content="A custom description of the blog post" />',
+        )
+
+    def test_templates_blogpost_detail_meta_description_excerpt(self):
+        """
+        The blogpost meta description should show the excerpt placeholder if no meta_description
+        placeholoder exists
+        """
+        blogpost = BlogPostFactory()
+        page = blogpost.extended_object
+
+        # Add an excerpt to the blogpost
+        placeholder = blogpost.extended_object.placeholders.get(slot="excerpt")
+        add_plugin(
+            language="en",
+            placeholder=placeholder,
+            plugin_type="PlainTextPlugin",
+            body="A further sub title of the blog post",
```

Comment:
Make it html to test that tags are stripped as well?

Reply:
The blogpost excerpt is uses the a plain text plugin. If I include html on the `body` it want be rendered (read and write) has html.